packages/react-next/src/components/Link/LinkBase.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { compose } from '@fluentui/react-compose';
+import { LinkProps } from './Link.types';
+import { useLink } from './useLink';
+
+export const LinkBase = compose<'a', LinkProps, LinkProps, {}, {}>(
```

Comment:
Unfortunately, we need to keep `KeytipData`. no breaking changes :)

Reply:
Done